ssl/ssl_stat.cc

```diff
@@ -113,11 +113,111 @@ const char *SSL_alert_type_string_long(int value) {
 }
 
 const char *SSL_alert_type_string(int value) {
-  return "!";
+  value >>= 8;
+  if (value == SSL3_AL_WARNING) {
+    return "W";
+  } else if (value == SSL3_AL_FATAL) {
+    return "F";
+  }
+
+  return "U";
```

Comment:
Should we to mask out the bits we don't care about?
```
value &= 0x0f00;
value >>= 8;
```

Reply:
Sure, might as well make the improvement :)